src/ert/ensemble_evaluator/_builder/_function_task.py

```diff
@@ -104,11 +104,11 @@ def run_job(
                 ev_data={ids.ERROR_MSG: str(e)},
             )
             raise e
-        else:
```

Comment:
Should be fine, but not related to this PR? Suggest splitting to seperate commit.